From c229e992230dd9ca2460b8ae4a31e36a26820a1d Mon Sep 17 00:00:00 2001
From: Mario Limonciello <mario.limonciello@dell.com>
Date: Wed, 10 Mar 2021 16:07:48 -0600
Subject: [PATCH] fu-tool: fix a regression where activate stopped working in
 c7d870aa9

Activate was no longer calling coldplug meaning even if devices were
in the pending database they wouldn't activate.
---
 src/fu-tool.c | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/src/fu-tool.c b/src/fu-tool.c
index 64ac0f36..75912616 100644
--- a/src/fu-tool.c
+++ b/src/fu-tool.c
@@ -1549,7 +1549,12 @@ fu_util_activate (FuUtilPrivate *priv, gchar **values, GError **error)
 		return FALSE;
 
 	/* load engine */
-	if (!fu_util_start_engine (priv, FU_ENGINE_LOAD_FLAG_READONLY, error))
+	if (!fu_util_start_engine (priv,
+				   FU_ENGINE_LOAD_FLAG_READONLY |
+				   FU_ENGINE_LOAD_FLAG_COLDPLUG |
+				   FU_ENGINE_LOAD_FLAG_REMOTES |
+				   FU_ENGINE_LOAD_FLAG_HWINFO,
+				   error))
 		return FALSE;
 
 	/* parse arguments */
-- 
2.25.1

